src/logic/Operate.js

```diff
@@ -0,0 +1,31 @@
+/*eslint-disable */
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
```

Comment:
- [ ] You have a typo on the `Operate.js` file name the first letter is supposed to be lowercase instead of uppercase to make the app compile without error, So please rename it to make the app work properly.

![image](https://user-images.githubusercontent.com/68796232/189346037-349fa97a-a814-42e3-9ee1-d35ca2497ef4.png)